src/plasma/plasma_store.cc

```diff
@@ -650,6 +676,7 @@ void send_notifications(event_loop *loop,
       }
     }
     num_processed += 1;
+    free(data);
```

Comment:
Can you document where the corresponding `malloc` happened? I believe it is in `create_object_info_buffer` within `push_notification`.